lib/ansible/modules/net_tools/nmcli.py

```diff
@@ -314,15 +314,13 @@
   tasks:
 
   - name: install needed network manager libs
-    yum:
-      name: '{{ item }}'
+    package:
+      name:
+        - NetworkManager-glib
+        - nm-connection-editor
```

Comment:
`libnm-qt-devel` is missing. Can you confirm if it is not required or so ?

Reply:
I did it on purpose. It's not required.

Reply:
Ok. Cool.